lib/environments/meteorEnvironment.js

```diff
@@ -308,6 +308,31 @@ export default {
     return moduleName;
   },
 
+  moduleSideEffectImports: ({ pathToCurrentFile, config }: Object): Array<string> => {
+    let basePath = '';
+    if (pathToCurrentFile.endsWith('.js')) {
+      basePath = pathToCurrentFile.slice(0, -3);
+    } else if (pathToCurrentFile.endsWith('.jsx')) {
+      basePath = pathToCurrentFile.slice(0, -4);
```

Comment:
We have a `stripFileExtensions` configuration option that should be used here instead of these hard-coded extensions.